react/Image.tsx

```diff
@@ -128,6 +135,15 @@ function Image(props: ImageProps) {
       style={imageDimensions}
       ref={imageRef}
       className={handles.imageElement}
+      {...(experimentalSetExplicitDimensions &&
+      !width?.toString().includes('%') &&
+      !height?.toString().includes('%') &&
+      (widthWithoutUnits || heightWithoutUnits)
```

Comment:
Maybe creating a const will clarify this conditional purpose

Reply:
Done!